### DISCUSSION TITLE
Can markers be placed in 2D?

### DISCUSSION THREAD

**Discussion by johnnyd710** [iTwin/itwinjs-core#952] [Q&A] [answered]
[IModeljs Markers](https://www.itwinjs.org/learning/frontend/markers/?term=markers) accept a Point3D in the constructor. Can Markers be placed in 2D drawings as well? If not, is there something similar that could work? Any code samples would be appreciated.

**Comment by pmconne** [comment, marked as answer]
Markers can be used in 2d views - just specify 0 for the point's Z component.
